### DISCUSSION TITLE
Sharing links localhost:1313

### DISCUSSION THREAD

**Discussion by 0r1k** [nunocoracao/blowfish#1523] [Q&A]
![Screenshot from 2024-06-06 22-14-11](https://github.com/nunocoracao/blowfish/assets/99347708/a33def37-9be7-4b59-959d-76c7084726ab)
Hey, I got a problem.
In my shared links localhost:1313
Every place with "put your domain" has been filled
From where the value of the variable ?url is taken from

**Comment by nunocoracao** [comment]
Have you configured the baseurl?
